src/main/java/net/mcreator/ui/views/editor/image/ImageMakerView.java

```diff
@@ -207,31 +206,43 @@ public void save() {
 
 	public void saveAs() {

```

Comment:
There should be some space under the button and also enable realtime validation for the name input

Reply:
I already tried to add some space under the button and I never succeeded.

Reply:
Spacing should be made with empty border, not jemptybox where possible